server/src/dtos/user-preferences.dto.ts

```diff
@@ -16,6 +16,17 @@ class MemoryUpdate {
   enabled?: boolean;
 }
 
+class EmailNotificationsUpdate {
+  @ValidateBoolean({ optional: true })
+  enabled?: boolean;
+
+  @ValidateBoolean({ optional: true })
+  albumInvite?: boolean;
+
+  @ValidateBoolean({ optional: true })
+  albumUpdate?: boolean;
```

Comment:
I'm wondering if we want it to be more of an object structure. So something like `{ album: { invite: true, update: false }, ... }`.

Reply:
It would be more on the preference's response, which is already structured somewhat similar

```json
    emailNotifications: {
      enabled: true,
      albumInvite: true,
      albumUpdate: true,
    },
```

Reply:
I saw that and I don't think it's structured similarly to my suggestion.

Reply:
I want to keep it similar to the event for ease of tracking. Additionally, you need the master `enabled` flag for ease of mass toggle